package.json

```diff
@@ -1,6 +1,10 @@
 {
+  "type": "datasource",
```

Comment:
I'm not sure why `package.json` should be changed.
I think `src/plugin.json` contains these information already.

Reply:
@sasaki77, thank you for pointing this out. A correction has been pushed.

Reply:
I'm sorry my ambiguous comment.
I think these lines are also unnecessary.

```
  "backend": true,
  "alerting": true,
  "executable": "gpx_archiver-datasource-backend",
```

What do you think?

Reply:
I agree